### PR TITLE
User preferences delete user

### PR DESCRIPTION
**Pull request by Sylv11**

### CODE REVIEW

File: src/components/preferences/DeleteModal.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    return (
+        <Modal isOpen={props.open} toggle={props.toggleDeleteModal} className="user-modal">
+            <ModalHeader style={{background: 'white'}}>
+                Suppression de mon compte
+            </ModalHeader>
+            <ModalBody style={{background: 'white'}}>
+                {!error ?
+                    <div>
+                        Êtes-vous sûr de vouloir supprimer votre compte ?
+                        <br/><br/>
```
Comment (jasonvanmalder) [inline src/components/preferences/DeleteModal.js]: YO WHAT THE FUCK IS  THIS `<br/>` IS SHIT MAN USE `<p>YOUR TEXT HERE</p>`.

File: src/components/preferences/DeleteModal.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                        En supprimant votre compte, vous perdez l'accès à celui-ci et cela entraine la suppression de toutes les données vous concernant (publications, commentaires, fichiers, messages, etc).
+                        <br/>
```
Comment (jasonvanmalder) [inline src/components/preferences/DeleteModal.js]: SAME

File: src/components/preferences/DeleteModal.js (hunk 1 above)
Comment (Sylv11) [reply src/components/preferences/DeleteModal.js]: I WAS SICK

File: src/components/preferences/DeleteModal.js (hunk 2 above)
Comment (Sylv11) [reply src/components/preferences/DeleteModal.js]: SAME

**Pull request merged**